clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/DefaultOrchestrationProcessor.groovy

```diff
@@ -43,12 +43,17 @@ class DefaultOrchestrationProcessor implements OrchestrationProcessor {
   @Autowired
   Registry registry
 
-  Task process(List<AtomicOperation> atomicOperations) {
+  @Override
+  Task process(List<AtomicOperation> atomicOperations, String clientRequestId) {
 
     def orchestrationsId = registry.createId('orchestrations')
     def atomicOperationId = registry.createId('operations')
     def tasksId = registry.createId('tasks')
-    def task = taskRepository.create(TASK_PHASE, "Initializing Orchestration Task...")
+    def existingTask = taskRepository.getByClientRequestId(clientRequestId)
+    if (existingTask) {
+      return existingTask
+    }
```

Comment:
@cfieber @ajordens @robfletcher these three lines are new - actually running the code revealed that we'd still try to process each of the operations without this, which kind of defeats the purpose of the whole thing.